src/output-json-dns.c

```diff
@@ -286,6 +287,8 @@ static int JsonDoh2Logger(ThreadVars *tv, void *thread_data, const Packet *p, Fl
     LogDnsLogThread *td = (LogDnsLogThread *)thread_data;
     LogDnsFileCtx *dnslog_ctx = td->dnslog_ctx;
 
+    /* DOH2 is always logged in flow direction, as its driven by the scope of an
+     * HTTP transation */
```

Comment:
@catenacyber Can you confirm? In our S-V tests we have no "event_type" of "dns" with just a request.

Reply:
More of a curiosity at this point:
- Does DoH echo the request back in the response like normal DNS?
- If the echo'd back request happens to be different than that of the actual request, which one is logged?
I'm trying to prepare a documentation section on DNS with some implementation details of what to expect. Differences between DNS and DoH seems important to cover.